### DISCUSSION TITLE
Application opening browser

### DISCUSSION THREAD

**Discussion by mhoude1966** [winapps-org/winapps#90] [General]
Hi,

If the winapp application is opening a browser, will it shows remotely as well or not ?

Thanks

**Comment by KernelGhost** [comment]
Could you please provide more details on your question? Specifically, what do you mean by "shows remotely"?

**Comment by mhoude1966** [comment]
I meant, if the winapps application is opening a web browser, will I see it
?

Thanks

**Comment by KernelGhost** [comment]
WinApps operates by creating an RDP connection to a Windows system using the provided user credentials. When you request to open a web browser, the browser window will be forwarded and displayed on your local system through FreeRDP and X11. This browser window should not be simultaneously visible anywhere else.
